src/dlgProfilePreferences.cpp

```diff
@@ -64,20 +63,25 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
     // should be added to the (QGridLayout*) returned by:
     // qobject_cast<QGridLayout*>(groupBox_debug->layout())
 
-    QFile file_use_smallscreen(mudlet::getMudletPath(mudlet::mainDataItemPath, QStringLiteral("mudlet_option_use_smallscreen")));
-    checkBox_USE_SMALL_SCREEN->setChecked(file_use_smallscreen.exists());
-    checkBox_showSpacesAndTabs->setChecked(mudlet::self()->mEditorTextOptions & QTextOption::ShowTabsAndSpaces);
-    checkBox_showLineFeedsAndParagraphs->setChecked(mudlet::self()->mEditorTextOptions & QTextOption::ShowLineAndParagraphSeparators);
-    // As we reflect the state of the above two checkboxes in the preview widget
-    // on another tab we have to track their changes in state and update that
-    // edbee widget straight away - however we do not need to update any open
-    // widgets of the same sort in use in ANY profile's editor until we hit
-    // the save button...
-    checkBox_reportMapIssuesOnScreen->setChecked(mudlet::self()->showMapAuditErrors());
-
-    MainIconSize->setValue(mudlet::self()->mToolbarIconSize);
-    TEFolderIconSize->setValue(mudlet::self()->mEditorTreeWidgetIconSize);
-    switch (mudlet::self()->menuBarVisibility()) {
+    mudlet* pMudlet = mudlet::self();
+
+    checkBox_USE_SMALL_SCREEN->setChecked(pMudlet->mEnableFullScreenMode);
+
+    // As we reflect the state of these next two checkboxes in the preview
+    // widget on another tab we have to track their changes in state and update
+    // that edbee widget straight away and as we can now have multiple profiles
+    // each with their own instance of this form open we have to update any open
+    // widgets of the same sort in use in ANY profile's editor immediately.
+    checkBox_showSpacesAndTabs->setChecked(pMudlet->mEditorTextOptions & QTextOption::ShowTabsAndSpaces);
+    checkBox_showLineFeedsAndParagraphs->setChecked(pMudlet->mEditorTextOptions & QTextOption::ShowLineAndParagraphSeparators);
+
+    checkBox_reportMapIssuesOnScreen->setChecked(pMudlet->showMapAuditErrors());
+
+
+    MainIconSize->setValue(pMudlet->mToolbarIconSize);
+    TEFolderIconSize->setValue(pMudlet->mEditorTreeWidgetIconSize);
```

Comment:
What is a "TE folder"?

Reply:
I suspect it is short for *Trigger Editor* - but I didn't create that variable name so I can't be blamed for it...! It is (in pixels when multiplied by 8) the icon size used in the treeWidget used in the left hand side of the Editor.